modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -123,6 +125,7 @@ public class RaftGroupServiceImpl implements RaftGroupService {
      * @param membersConfiguration Raft members configuration.
      * @param leader Group leader.
      * @param executor Executor for retrying requests.
+     * @param commandsMarshaller Marshaller that should be used to [de]serialize commands.
```

Comment:
```suggestion
     * @param commandsMarshaller Marshaller that should be used to de]serialize commands.
```

Reply:
Hm? I'll change the wording if brackets confuse you

Reply:
It would be great, but at your discretion.